tests/unit/inference/v2/modules/test_quantized_linear_module.py

```diff
@@ -145,10 +145,9 @@ def _fp6_quantized_linear_helper(tokens: int,
     ActivationType.ReGLU,
     ActivationType.SiGLU,
 ]
-all_tokens = [1, 37]
```

Comment:
Do we just not support these cases anymore or should we make them conditional on the hardware they are running on?

Reply:
This is to reduce the number of tests for FP6 kernel. Currently, the UTs of FP6 kernels are too time consuming.

Reply:
Sounds good, thanks!